MarkupsToModel/MRML/vtkMRMLMarkupsToModelNode.cxx

```diff
@@ -159,6 +164,19 @@ void vtkMRMLMarkupsToModelNode::ReadXMLAttributes( const char** atts )
       }
       SetInterpolationType(interpolationType);
     }
+    else if ( ! strcmp( attName, "PointParameterType" ) )
+    {
+      int pointParameterType = 0;
+      std::stringstream nameString;
+      nameString << attValue;
+      nameString >> pointParameterType;
```

Comment:
I do understand why we want to do this, and I will store enum strings instead of ints in the settings file.  I assume I need to make changes to InterpolationType too (?)